### PR TITLE
uk to us fixups

### PR DESCRIPTION
**Pull request by axtens**

### CODE REVIEW

File: building/configlet/README.md.new [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Configlet
```
Comment (iHiD) [inline building/configlet/README.md.new]: @axtens I think this file got added by mistake.

**Comment by iHiD** [comment]
This looks great. Thanks. I've left one comment that needs resolving :)

File: building/configlet/README.md.new (hunk 1 above)
Comment (axtens) [reply building/configlet/README.md.new]: Definitely. An aspell artifact that should have been deleted

Comment (iHiD) [reply building/configlet/README.md.new]: Can you push an extra commit with it removed please?

File: building/markdown/markdown.md.new [hunk 2]
```diff
@@ -0,0 +1,168 @@
+# Markdown Specification
```
Comment (ee7) [inline building/markdown/markdown.md.new]: > I think this file got added by mistake.

(And this file?)

File: building/tracks/config-json.md [hunk 3]
```diff
@@ -238,7 +238,7 @@ Each concept is an entry in the top-level `concepts` array. The following fields
 ## Key features
 
 The language's key features succinctly describe what the most important features of the language are.
-They are intended to upsell the more interesting features of a language to potential students.
+They are intended to outsell the more interesting features of a language to potential students.
```
Comment (ee7) [inline building/tracks/config-json.md]:
```suggestion
They are intended to promote the more interesting features of a language to potential students.
```

Maybe "promote" or "advertize" here.

["outsell" is incorrect](https://www.merriam-webster.com/dictionary/outsell):

> Definition of outsell
> transitive verb

> 1: archaic : to exceed in value
> 2: to exceed in number of items sold
> 3: to surpass in selling or salesmanship

> Examples of outsell in a sentence:
> - Diet sodas are now beginning to outsell regular sodas in the supermarkets.
> - We've been outselling our competitors for the past five years.


["upsell" was probably incorrect too](https://www.merriam-webster.com/dictionary/upsell):
> Definition of upsell
> transitive verb

> to try to convince (a customer) to purchase something additional or at a higher cost

> Examples of upsell in a sentence:
> - With a strong correlation between the amount of video consumed and the Internet speed that consumers think they need, providers could possibly upsell customers on costlier broadband packages.
> - They start upselling you before you book your vacation by offering upgrades to a higher-class cabin

File: building/configlet/README.md.new (hunk 1 above)
Comment (axtens) [reply building/configlet/README.md.new]: I have tried to do that. I'm not at all sure if successfully.

**Comment by SleeplessByte** [comment]
Looking great now!

**Review by ee7 — APPROVED** [review]
Yes, it looks correct with 574234b.

See the diff between initial commit and final commit: https://github.com/exercism/docs/compare/358e673ac7292df7bb9a91e392047ead675413fc...574234b9ebf1fc7ee2d536dadde6bb813615762a

**Comment by iHiD** [comment]
Great. Thank you for the work @axtens and everyone else for the help/reviews! :)

**Pull request merged**